MVC/controller.js

```diff
@@ -81,12 +81,28 @@ const patchArticleById = (req, res, next) => {
     })
 }
 
+const deleteCommentsById = (req, res, next) => {
+    const { comment_id } = req.params
+
+    checkValidCommentId(req.params.comment_id)
+    .then(() => {
+        return removeCommentsById(comment_id)
+    })
+    .then((comment) => {
+        res.status(204).send({ comment: comment })
+    })
+    .catch((err) => {
+        next(err)
+    })
```

Comment:
Good use of util function checkValidCommentId. Throwing a customs error in the model might be even easier